_wiki/FAQ/General.md

```diff
@@ -55,15 +52,11 @@ This can be performed from the UI or calculated manually.
 Check the configuration file for your tablet [here](https://github.com/OpenTabletDriver/OpenTabletDriver/tree/master/OpenTabletDriver.Configurations)
 or within the driver for properly calculated digitizer dimensions in millimeters.
 
-{:.text-muted .px-3 .mb-3}
-<small>
-**Note:** Huion and Gaomon areas use a "percentage area" , which uses a percentage
+> Note: Huion and Gaomon areas use a "percentage area" , which uses a percentage
 of the tablet's maximum area to calculate the area.\\
-  You must know your tablet's digitizer dimensions in millimeters in order to properly convert these areas.\\
-  This is automatically handled in the UI, for the best results it is recommended to use it for converting from these proprietary drivers.
-</small>
+> You must know your tablet's digitizer dimensions in millimeters in order to properly convert these areas.
```

Comment:
```suggestion
> You must know your tablet's digitizer dimensions in millimeters in order to properly convert these areas.\\
```

otherwise this drops the newline

Reply:
When there's consecutive blockquotes, markdown renderers concatenate their content without newline. Adding the `\\` somehow spawns a newline. The `\\` above this line is actually unintended as I want blockquotes be just oneliners when rendered. I may split up the other long blockquote in review to this style if that's preferred.

Reply:
We can just do it in linting if lines go on for too long (just-opened: #30) It's mostly a Markdown readability concern I have, not so much site readability since as you said it should probably just be split into multiple lines if it is hard to read.